### PR TITLE
Return CreateEntryResult for import doc api with an invalid parent ID

### PR DESCRIPTION
**Pull request by steven-xu-lf**
- Return `CreateEntryResult` type of response for an import document request with an invalid parent ID
- Added 2 additional integration tests
- Updated changelog
- Updated pom.xml file
- Ensured that the other import doc test cases were still running

### CODE REVIEW

File: src/main/java/com/laserfiche/repository/api/clients/impl/EntriesClientImpl.java [hunk 1]
```diff
@@ -198,21 +198,19 @@ public CompletableFuture<CreateEntryResult> importDocument(String repoId, Intege
...
-                            return null;
+                    String jsonString = new JSONObject(body).toString();
+                    try {
+                        CreateEntryResult response = objectMapper.readValue(jsonString, CreateEntryResult.class);
+                        if (response.getOperations() == null) {
+                            throw new IllegalStateException();
                         }
-                    } else {
-                        ProblemDetails problemDetails;
+                        return objectMapper.readValue(jsonString, CreateEntryResult.class);
```
Comment (jason-chan-lf) [inline src/main/java/com/laserfiche/repository/api/clients/impl/EntriesClientImpl.java]: could return the `response` instead of calling `objectMapper.readValue` again

File: src/test/java/integration/ImportDocumentApiTest.java [hunk 2]
```diff
@@ -248,4 +248,49 @@ void importDocument_Returns_Detail_When_PartialSuccess_Happens() {
...
+                .getEntryCreate()
+                .getExceptions()
+                .get(0);
+        assertTrue(entryCreateException
+                .getMessage()
+                .startsWith("Entry not found."));
+        assertEquals(HttpURLConnection.HTTP_NOT_FOUND, entryCreateException.getStatusCode());
+        assertEquals(ErrorSource.LASERFICHE_SERVER.getName(), entryCreateException.getErrorSource());
+    }
+
```
Comment (jason-chan-lf) [inline src/test/java/integration/ImportDocumentApiTest.java:297]: could you also add a test that causes a ProblemDetails response to be returned? I saw none of the current import api tests test this case yet. It would be nice to know that the import api can handle this response type.

Comment (steven-xu-lf) [reply src/test/java/integration/ImportDocumentApiTest.java:297]: ok added a test case with an invalid repo as a part of the request

File: src/main/java/com/laserfiche/repository/api/clients/impl/EntriesClientImpl.java (hunk 1 above)
Comment (steven-xu-lf) [reply src/main/java/com/laserfiche/repository/api/clients/impl/EntriesClientImpl.java]: ok done

**Review by jason-chan-lf — APPROVED** [review]
